cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/InstancePricingDTO.java

```diff
@@ -39,41 +46,59 @@
     @ApiModelProperty( "Whether the pricing will not be linked to any Virtual Machine Configuration" )
     private Boolean deprecated;
 
+    public UrlEntityDTO getInstanceType()
+    {
+        return instanceType;
+    }
+
+    public void setInstanceType( UrlEntityDTO instanceType )
+    {
+        this.instanceType = instanceType;
+    }
+
+    @Deprecated
     public String getName()
     {
         return name;
     }
 
+    @Deprecated
     public void setName( String name )
     {
         this.name = name;
     }
 
+    @Deprecated
     public Integer getRam()
     {
         return ram;
     }
 
+    @Deprecated
     public void setRam( Integer ram )
     {
         this.ram = ram;
     }
 
+    @Deprecated
     public BigDecimal getCpu()
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the missing @deprecated Javadoc tag. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS1133)